### ISSUE TITLE
What is the best way to add HVAC system to current "ExportToOpenStudio"

### ISSUE THREAD

**Issue by MingboPeng** [mostaphaRoudsari/Honeybee#697]
Hi @mostaphaRoudsari  @chriswmackey ,

As you know, I am adding a bunch of hvac components to current HB workflow (HB+ as well when the energy part is added), and thinking what is the best way to merge the osm model that HB generated.

Here is a diagram of workflow:
![image](https://user-images.githubusercontent.com/9031066/41109438-0701fcde-6a45-11e8-8509-c76d12e32ea5.png)

Solution1:
Keep everything still, and add all hvac components right after saved osm file before run. (I can add a merge method in C#, so in ExportToOpenStudio, just need to call it).

Solution 2:
use the openstudio model from hvac system as starter, instead of an empty osm model. But this require a bit works to check and connect ThermalZone to Spaces 

Please let me know if this makes sense.
Thanks,
Mingbo

**Comment by chriswmackey** [comment]
@MingboPeng ,
It's hard to give advice on the best route forward without knowing exactly what you want to customize in the HVAC.  I can tell you that the way to implement the first one is very straightforward.  You just export the conditioned zones with the thermostat only and then you can add a fully detailed HVAC after the export.  For the second one, you'd likely have to edit the HVAC-setting scripts in the Export to OpenStudio component.

**Comment by MingboPeng** [comment]
Agreed, the first one changes the minimum of "ExportToOpenstudio". This might be easier to maintain later.
